### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by austintraver**

### CODE REVIEW

**Review by NessDan — APPROVED** [review]
Haha was just about to make a PR to fix the "genetics" vs "gentics" typo!

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Halmak Keyboard Layout
 
 This is an AI designed keyboard layout that was built within the
-[keyboard-gentics](https://github.com/MadRabbit/keyboard-genetics) project.
+[keyboard-genetics][] project.
```
Comment (dessalines) [inline README.md:4]: What are those extra square brackets on the end for? you don't need those.

Comment (NessDan) [reply README.md:4]: If you scroll to the bottom of the changes, I think this is a way for you to specify that the text is a link (e.g. "keyboard-genetics") but you specify where it links to at the bottom of the file [(here's where they do that.)][]

You can even do it yourself on a Github comment if you'd like to test it out like I did

image

[(here's where they do that.)]: https://github.com/MadRabbit/halmak/pull/28/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R91

Comment (dessalines) [reply README.md:4]: You don't need extra brackets for [that]

[that]: https://ddg.gg

Standard markdown is just:

```
You don't need extra brackets for [that]

[that]: https://ddg.gg
```

Comment (austintraver) [reply README.md:4]: TIL! Thank you for sharing 🙏🏻

Comment (NessDan) [reply README.md:4]:
```suggestion
[keyboard-genetics] project.
```

File: README.md [hunk 2]
```diff
@@ -27,11 +27,11 @@ During the research I've identified the following results: in terms of efficienc
 * `Workman` - `+101%`
 * `Halmak` - `+134%`
 
-Please refer to [this article](http://nikolay.rocks/2016-12-20-the-halmak-reborn)
+Please refer to the article *[Halmak Reborn][]*
```
Comment (NessDan) [inline README.md:30]:
```suggestion
Please refer to the article *[Halmak Reborn]*
```

File: README.md [hunk 3]
```diff
@@ -27,11 +27,11 @@ During the research I've identified the following results: in terms of efficienc
...
 for the more detailed breakdown of the results.
 
 All scripts are opensourced and can be verified in the
-[keyboard-gentics](https://github.com/MadRabbit/keyboard-genetics) repository.
+[keyboard-genetics][] repository.
```
Comment (NessDan) [inline README.md:34]:
```suggestion
[keyboard-genetics] repository.
```

**Pull request closed without merging**